src/site/site.xml

```diff
@@ -40,45 +40,30 @@
 
   <body>
 
-  <links>
-            <item
-                    name="Home"
-                    href="http://james.apache.org/index.html" />
-            <item
-                    name="Server"
-                    href="http://james.apache.org/server/index.html" />
-            <item
-                    name="Hupa"
-                    href="http://james.apache.org/hupa/index.html" />
-            <item
-                    name="Protocols"
-                    href="http://james.apache.org/protocols/index.html" />
-            <item
-                    name="Mailets"
-                    href="http://james.apache.org/mailet/index.html" />
+
+        <links>
             <item
-                    name="Mailbox"
-                    href="http://james.apache.org/mailbox/index.html" />
+                name="Home"
```

Comment:
Home is the button leading to James Marketing website and James leads to all the internal james project